From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Viljanen <matti.viljanen@jolla.com>
Date: Wed, 8 Oct 2025 22:12:27 +0300
Subject: [PATCH] Dilute user and group requires into recommends

The user and group dependencies are hard requirements with rpm 4.19.
Generating the `Provides: group(foo)` and such is done automatically
when done using a supported method. We have currently several packages
not doing this correctly, so we have to loosen the setting for now.

https://rpm.org/docs/4.19.x/manual/users_and_groups.html
---
 macros.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/macros.in b/macros.in
index 4cd8f98a7..934e25148 100644
--- a/macros.in
+++ b/macros.in
@@ -244,7 +244,7 @@ Supplements:   (%{name} = %{version}-%{release} and langpacks-%{1})\
 %clamp_mtime_to_source_date_epoch 0
 
 #	If enabled, dilute user() and group() requires into recommends
-#%_use_weak_usergroup_deps 1
+%_use_weak_usergroup_deps 1
 
 #	The directory where newly built binary packages will be written.
 %_rpmdir		%{_topdir}/RPMS
